src/xcode/ENA/ENA/Source/Scenes/RiskLegend/Cell/RiskLegendDotBodyCell.swift

```diff
@@ -6,6 +6,36 @@ import UIKit
 
 class RiskLegendDotBodyCell: UITableViewCell {
 
-	@IBOutlet weak var dotView: UIView!
+	var dotView: UIView!
+	var label: ENALabel!
 	
+	override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
```

Comment:
please add the MARKS from the project here and move init / var to the matching sections.

Reply:
@KaiTeuber Which MARKS? Is there a documentation where I can see the sections?

Reply:
Down here in the readme section is a link to a code snipet https://github.com/corona-warn-app/cwa-app-ios